assets/app/view/game/spreadsheet.rb

```diff
@@ -222,7 +222,7 @@ def render_corporation(corporation, operating_order, current_round)
             h(:td, sold_props, p.num_shares_of(corporation).to_s + (corporation.president?(p) ? '*' : ''))
           end,
           h(:td, corporation.num_shares_of(corporation)),
-          h(:td, @game.share_pool.num_shares_of(corporation)),
+          h(:td, @game.share_pool.num_shares_of(corporation).to_s + (corporation.owner == @game.share_pool ? '*' : '')),
```

Comment:
owenr.share_pool?

Reply:
or owner.game?

Reply:
or owner.bank?

Reply:
no, it should be called corporation.receivership?

Reply:
I switched to the method `pool?` in 75b60a63.

At first I tried bank, but switched to share pool since the share pool owns the actual shares, IIRC that simplified some things (I don't seem to have any specific notes on this point and it's been a while now since I made that change). `process_bankrupt` sets the corporation's owner so we could set it to whatever we want.

Is receivership a term used in other games? Don't think I've come across that one.

Reply:
yes, it's very common, if a corporation has no president, it is considered in receivership,

rolling stock, 1860 etc let's change to receivership method

Reply:
e9141258